### PR TITLE
Vue demo

### PR DESCRIPTION
**Pull request by kyler-rosquist**
The great majority of the 271 changes are `asset` files, which really bloat the project. The actual Vue code is quite lean.

I would recommend using the "files filter" to remove `svg`s, etc... when viewing the files changed. I still have a small bit of clean up work left.

### CODE REVIEW

File: vue/src/store/sdk.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type { SDKInputField } from '@/types';
+import Portfolio from '@youngalfred/bowtie-sdk';
+import { computed, reactive } from 'vue'
+
+// const app = reactive<{portfolio: Portfolio, updateField: (a: string) => (b: string) => void}>({
```
Comment (idreamer) [inline vue/src/store/sdk.ts:5]: Dead code?

File: vue/src/components/fieldgroups/handlers.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  'select': Select,
+  'radio': Radio,
+  'check': Check,
+  'file': null,
+  'fieldgroup': Fieldgroup
+}
+
+// prettier-ignore
+export const CLASS_HANDLERS: Record<string, any> = {
+  // 'multi-select-dropdown': LabeledField(PoppedField(MultiSelect)),
```
Comment (idreamer) [inline vue/src/components/fieldgroups/handlers.ts:44]: Dead codes?

File: vue/src/components/fieldgroups/GridGroup.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        field: {
+            type: Object as PropType<Fieldgroup>,
+            required: true
+        }
+    })
+
+</script>
+
+<template>
+    <!-- :style="{ 'grid-template-columns': `repeat(${field.children.length}, minmax(125px, 175px))` }" -->
```
Comment (idreamer) [inline vue/src/components/fieldgroups/GridGroup.vue:16]: Let's clean up...

File: vue/src/components/fieldgroups/modifiers/field-builder.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        // Do not render prefilled or hidden fields
+        if (child.kind === "hidden") {
+          return acc;
+        }
+    
+        const { kind, children = [], valid: { valid, msg: warning }, ...groupRest } = child as SDKFieldGroup;
+        // reduce the multigroup/fieldgroup's children (they need an onChange event handler and stringified classes)
+        if (["multigroup", "fieldgroup"].includes(kind)) {
+          groupDecorations = DECORATORS[groupRest.id] || groupDecorations
+          // console.log({ id: groupRest.id, decorationsSize: Object.keys(groupDecorations).length })
```
Comment (idreamer) [inline vue/src/components/fieldgroups/modifiers/field-builder.ts:25]: Let's clean up...

File: vue/src/components/fieldgroups/modifiers/field-builder.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            })];
+        }
+    
+        const { id, ...rest } = child as SDKInputField;
+
+        if (rest.kind === 'hidden') {
+            throw new Error('Unexpected type hidden. Should have been filtered out already.')
+        }
+
+        // console.log({groupDecorations, id: id.split('.').pop(), fullid: id })
```
Comment (idreamer) [inline vue/src/components/fieldgroups/modifiers/field-builder.ts:46]: Let's clean up...

**Review by idreamer — APPROVED** [review]

**Comment by kyler-rosquist** [comment]
This PR was closed in favor of: https://github.com/youngalfred/bowtie-sdk/pull/17, which is an improvement of the same code.

**Pull request closed without merging**